translations/en.json

```diff
@@ -2988,7 +2988,7 @@
   "what-to-do": "What you would like to do?",
   "what-we-will-receive": "What we will receive:",
   "what-we-wont-receive": "What we won't receive:",
-  "what-you-can-do": "What you can do?",
+  "what-you-can-do": "What can you do?",
```

Comment:
Hi @flexsurfer, Thank you for the PR.
Please can you share link related to this change.

Reply:
there will be more in this PR